### PR TITLE
fix: (Platform) add fixes for Search Field defects

### LINKED ISSUE #3979: Bug: (Platform) Search Field – search icon looks weird
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
Icon looks weird:
image

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
fundamental-ngx: v 0.25.0
#### Is there anything else we should know?
Chrome/Safari
upd: duplicate https://github.com/SAP/fundamental-ngx/issues/3773

### LINKED ISSUE #3980: Bug: (Platform) Search Field – dropdown with suggestions do not appear
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
Dropdown with suggestions do not appear: 
search_suggestions

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
fundamental-ngx: v 0.25.0
#### Is there anything else we should know?
Chrome/Safari

### PR DESCRIPTION
**Pull request by KevinOkamoto**
#### Please provide a link to the associated issue.

#3979
#3980 
#3981

#### Please provide a brief summary of this pull request.

* Refactored button icons in accordance to latest fundamental-styles (fixes #3979)
* Added ScrollingModule to SearchField documentation module (fixes #3980 and #3981)

Screen Shot 2020-12-02 at 10 21 59 AM

Screen Shot 2020-12-02 at 10 22 22 AM

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

### CODE REVIEW

**Comment by kavya-b** [comment]
I think #3993 also addresses the icons issue :)

**Review by olshansky — APPROVED** [review]

File: apps/docs/src/app/platform/component-docs/platform-search-field/platform-search-field-examples/platform-search-field-basic-example.component.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import "~@angular/cdk/overlay-prebuilt.css";
```
Comment (Betrozov) [inline apps/docs/src/app/platform/component-docs/platform-search-field/platform-search-field-examples/platform-search-field-basic-example.component.scss:1]: please remove it
already fixed in this [PR](https://github.com/SAP/fundamental-ngx/pull/3985)

File: apps/docs/src/app/platform/component-docs/platform-search-field/platform-search-field-examples/platform-search-field-categories-example.component.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@import "~@angular/cdk/overlay-prebuilt.css";
```
Comment (Betrozov) [inline apps/docs/src/app/platform/component-docs/platform-search-field/platform-search-field-examples/platform-search-field-categories-example.component.scss:1]: please remove it

File: apps/docs/src/app/platform/component-docs/platform-search-field/platform-search-field-examples/platform-search-field-data-source-example.component.scss [hunk 3]
```diff
@@ -1,3 +1,4 @@
+@import "~@angular/cdk/overlay-prebuilt.css";
```
Comment (Betrozov) [inline apps/docs/src/app/platform/component-docs/platform-search-field/platform-search-field-examples/platform-search-field-data-source-example.component.scss:1]: please remove it

File: libs/platform/src/lib/components/search-field/search-field.component.html [hunk 4]
```diff
@@ -23,21 +23,27 @@
             [ngClass]="{ 'fd-input-group__addon--compact': contentDensity === 'compact' }"
             *ngIf="inputText && inputText.length > 0">
             <button
-                class="fdp-search-field__clear fd-input-group__button fd-button fd-button--transparent sap-icon--decline"
-                [ngClass]="{ 'fd-button--compact': contentDensity === 'compact' }" (click)="clearTextInput()"></button>
+                class="fdp-search-field__clear fd-input-group__button fd-button fd-button--icon fd-button--transparent"
+                [ngClass]="{ 'fd-button--compact': contentDensity === 'compact' }" (click)="clearTextInput()">
+                <i class="sap-icon--decline"></i>
```
Comment (Betrozov) [inline libs/platform/src/lib/components/search-field/search-field.component.html:28]: already fixed in this [PR](https://github.com/SAP/fundamental-ngx/pull/3993)

**Review by Betrozov — CHANGES_REQUESTED** [review]

**Comment by KevinOkamoto** [comment]
Closing this PR as it has already been addressed in PRs #3993 and #3985.

**Pull request closed without merging**